v3/client.go

```diff
@@ -9,30 +9,41 @@ import (
 	"fmt"
 	"net/http"
 	"runtime"
-	"strings"
 	"time"
 
 	api "github.com/exoscale/egoscale/v3/api"
 	"github.com/exoscale/egoscale/version"
 )
 
-type ClientZone string
+// URL represents a zoned url endpoint.
+type URL string
 
 const (
-	ClientZoneCHGva2 ClientZone = "ch-gva-2"
-	ClientZoneCHDk2  ClientZone = "ch-dk-2"
-	ClientZoneDEFra1 ClientZone = "de-fra-1"
-	ClientZoneDEMuc1 ClientZone = "de-muc-1"
-	ClientZoneATVie1 ClientZone = "at-vie-1"
-	ClientZoneATVie2 ClientZone = "at-vie-2"
-	ClientZoneBGSof1 ClientZone = "bg-sof-1"
+	CHGva2 URL = "https://api-ch-gva-2.exoscale.com/v2"
+	CHDk2  URL = "https://api-ch-dk-2.exoscale.com/v2"
+	DEFra1 URL = "https://api-de-fra-1.exoscale.com/v2"
+	DEMuc1 URL = "https://api-de-muc-1.exoscale.com/v2"
+	ATVie1 URL = "https://api-at-vie-1.exoscale.com/v2"
+	ATVie2 URL = "https://api-at-vie-2.exoscale.com/v2"
+	BGSof1 URL = "https://api-bg-sof-1.exoscale.com/v2"
 )
 
+// URLs represents a list of zoned URL.
+var URLs map[string]URL = map[string]URL{
```

Comment:
```suggestion
var Zones map[string]URL = map[string]URL{
```
Maybe this is clearer for the variable name